event_routing_backends/helpers.py

```diff
@@ -149,9 +149,8 @@ def get_course_from_id(course_id):
     course_overviews = get_course_overviews([course_key])
     if course_overviews:
         return course_overviews[0]
-    return {
-        "display_name": "Unknown Course",
-    }
+    raise ValueError(f"Course with id {course_id} does not exist.")
```

Comment:
Can't remember, will this be caught upstream so it doesn't break bulk export?

Reply:
Not sure

Reply:
The only helper that raises an exception is `get_anonymous_user_id` and it raises a `ValueError` exception. It shouldn't break the workflow.

@ziafazal Am I right?

Reply:
@Ian2012 yes `ValueError` is caught [here](https://github.com/openedx/event-routing-backends/blob/99ede4ede3f71a8769b122b9ec8db50766b8a71d/event_routing_backends/backends/events_router.py#L83) so it should not break.